models/Game.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
+
+const gameSchema = new mongoose.Schema({
+  date: {
+    type: Date,
+    require: true,
```

Comment:
```suggestion
    required: true,
```
required로 고쳐야할 것 같아요!